### DISCUSSION TITLE
Copy link to block

### DISCUSSION THREAD

**Discussion by jsp-kld** [toeverything/AFFiNE#6346] [Ideas]
### Description

Notion has the "Copy link to block" menu, so that users can get the block-level URL. Will AFFiNE support this feature? 
![Pasted image 20240327121414](https://github.com/toeverything/AFFiNE/assets/119592769/52b6ba05-a341-4072-86ae-c20d8aa15e73)

### Use case

This is very useful when I am sharing a paragraph of a long article, others will get to the block directly. 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by jshpng** [comment]
This should be a fundamental feature for a block-level note-taking tool. 
Block-level bidirectional link is also important.

**Comment by HugoVilardouro** [comment]
The ability to use the generated URL in any link is fundamental to make this useful.
What I want is to:
- go to page A;
- select a block and copy an link to block
- go to page B
- select some text, and covert that text to a link that will take me to the block on page A
